api/app/signals/apps/signals/management/commands/sia_stadsdeel.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management import BaseCommand
```

Comment:
Not blocker:

You could add one or two line docstring explaining the contents of this module / command.